pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -182,10 +182,17 @@ func RunStartBuild(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra
 		env = append(env, kapi.EnvVar{Name: "BUILD_LOGLEVEL", Value: buildLogLevel})
 	}
 
-	request := &buildapi.BuildRequest{
-		ObjectMeta: kapi.ObjectMeta{Name: name},
-	}
-	if len(env) > 0 {
+  buildCauses := []*buildapi.BuildTriggerCauses{}
```

Comment:
most of what was done here, is in conformity with the comments made for deployment trigger information that was implemented here: https://github.com/openshift/origin/pull/302#discussion_r19691430